Assets/CardManager/Card.cs

```diff
@@ -69,14 +69,26 @@ private void HandleSpawning()
         List<Tile> tileList = arena.getTileList();
         Tile tile = tileList.Find(obj => obj.isClicked);
 
-        if (!unitSpawn.Spawn(tile, this.cardDetails, power, arena.playerTurn, model, index))
+        // if the card spawns unit (spell card can be cast on enemy units outside of your range / frontline
+        if (true)
```

Comment:
if true?

Reply:
jak spelle będą (następna rzecz za którą chciałem się brać to będzie if karta spawni jednostkę